host-interaction/process/inject/allocate-rw-memory.yml

```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: allocate RW memory
+    namespace: host-interaction/process/inject
+    author: 0x534a@mailbox.org
+    scope: basic block
+    att&ck:
+      - Defense Evasion::Process Injection [T1055]
```

Comment:
Propose to remove this as not specific enough to process injection?

Reply:
Fixed in aa94a33